faasmcli/faasmcli/util/wasm.py

```diff
@@ -0,0 +1,1104 @@
+import warnings
+warnings.filterwarnings("ignore", category=DeprecationWarning) 
+from faasmcli.util.env import WASM_OPT_BIN
+import subprocess
+from subprocess import check_output
+import re
+import os
+import codecs
+import random
+import string
+def get_random_string(length=8) -> str:
+    """
+        credits: https://pynative.com/python-generate-random-string/
+    """
```

Comment:
Again, no need to credit blog posts and stack overflow for simple algorithms and use of the standard library